### DISCUSSION TITLE
G7 support (experimental) xDrip should not be used for medical (dosing) decisions

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#2778] [Show and tell]
No one should use xDrip for making medical (dosing) decisions. xDrip is not a medical app.
With that in mind, these are instructions you can follow:
https://navid200.github.io/xDrip/docs/Dexcom/G7.html

Please note that this is very new and could fail.  So, consider it experimental.

**Comment by fwolter** [comment]
Seems to work! The sensor started to deliver data nearly immediately after applying it. The values are way off (70mg/dl instead of 136mg/dl). I will try a calibration when the official max warm up time of 30min is exceeded.

The "Warm Up Time" display is stuck at 27min:
![grafik](https://user-images.githubusercontent.com/1894841/233659064-360b41b5-cf4f-4a8b-aa3a-ac39b6320f9c.png)

**Reply by Navid200** [reply]
Would you please tap on the red note?

**Reply by fwolter** [reply]
Done. The text "Tap to use nightly version" changed to "Tap to report", which I've done, too.

**Reply by Navid200** [reply]
@fwolter 
Do you mind contacting me privately?
This can be done by sending me (@Navid200 ) a Gitter message.
Alternatively, you can email me:  navid.pronouce.naveed at gmail dot com

**Comment by fwolter** [comment]
One thing I noticed is that the menu item "Initial Calibration" doesn't change to "Add Calibration" after doing an initial calibration.

**Reply by Navid200** [reply]
You can open a new discussion.

xDrip currently is not recognizing your device because of the firmware.  So, the behavior you describe is because of that.
It will be fixed soon.

**Reply by fwolter** [reply]
After applying #2792, the menu item behaves as expected.

**Reply by Navid200** [reply]
Would you please state that in a comment in that PR?  

This particular discussion you post in is not related to any particular firmware.
That PR is.

**Comment by fwolter** [comment]
I used the G7 until it expired without any further problems.
